lib/formatNumber.js

```diff
@@ -7,6 +7,9 @@
  * @param {*} number
  * @returns {string} 2900 -> '2 900'
  */
-export default function formatNumber(number) {
-  return Number(number).toString().replace(/(\d)(?=(\d{3})+([\D]|$))/g, '$1 ');
-}
+function formatNumber(number) {
+  return (typeof number === 'number'
+          ? Number(number.toPrecision(15)).toString()
```

Comment:
Please add comments for the function which make clear what are you doing here :)